.github/workflows/chrome-store-upload.yaml

```diff
@@ -0,0 +1,29 @@
+name: Upload To Chrome Web Store
+# This workflow is triggered on pushes to the repository.
```

Comment:
is this workflow triggered on pushes to the repo or when it detects changes to the version number?

Reply:
When a tag is pushed. I think you're hinting that we should release the large merge? :)